src/System.Private.ServiceModel/tests/Scenarios/Extensibility/WebSockets/WebSocketTests.cs

```diff
@@ -374,12 +387,11 @@ public static void WebSocket_Https_Duplex_TextStreamed()
 
     [Fact]
     [OuterLoop]
-#if !FEATURE_NETNATIVE
-    [ActiveIssue(4429)] // In #CoreFX
+#if FULLXUNIT_NOTSUPPORTED
+    [ActiveIssue(526)]
+#else
     [ActiveIssue(470)]
```

Comment:
470 should not apply to this test as 470 only applies to wss (https) while this test is http.